frontend/auth.ts

```diff
@@ -0,0 +1,7 @@
+
+import NextAuth from "next-auth"
+import Google from "next-auth/providers/google"
+ 
+export const { handlers, signIn, signOut, auth } = NextAuth({
+  providers: [Google],
+})
```

Comment:
```suggestion

})
```

nitpick: add new line at the end of the file which may follow the best practice
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file